### PR TITLE
GUT-126: build standalone VideoShowcase

### PR DESCRIPTION
**Pull request by ethitter**

### CODE REVIEW

**Comment by ethitter** [comment]
@laras126 This is ready for review! Two notes:

1. I set this up to put the new file in a `standalone` subdirectory to avoid potential naming conflicts, but that may not be necessary.
1. The version in `common.js` includes a call to the `ajaxIconSprite()`, but I wasn't sure if that was needed in this standalone version too.

File: package.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
     "larva": "packages/larva/bin/larva.js server --watch ./packages -e twig,js,html",
     "backstop": "backstop --config=node_modules/@penskemediacorp/backstopjs-config",
     "prod": "npm run prod:scss && npm run prod:js",
-    "prod:scss": "cd packages/larva-css && npm run build && cd ../../ && packages/larva/bin/larva.js prod-scss && npm run sync-build",
-    "prod:js": "packages/larva/bin/larva.js prod-js && npm run sync-build",
+    "prod:scss": "npm --prefix=packages/larva-css run build && packages/larva/bin/larva.js prod-scss && npm run sync-build",
```
Comment (laras126) [inline package.json:9]: I did not know about this!! Nice!

**Review by laras126 — APPROVED** [review]
Fabulous!

**Comment by laras126** [comment]
@ethitter Ah, I reviewed before seeing your other comment – 

> I set this up to put the new file in a standalone subdirectory to avoid potential naming conflicts, but that may not be necessary.

Not necessary for naming conflicts, but I like this because I could see wanting to have template-level bundles someday, and this seems like a nice clarification.

> The version in common.js includes a call to the ajaxIconSprite(), but I wasn't sure if that was needed in this standalone version too.

Correct not to include it here – we will need a generic `larva.js` that will include the icon sprite and font loading, and it will be enqueued by default for all post types.

**Pull request merged**